images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -103,7 +103,68 @@ _storage_overrides:
 
 meteringconfig_storage_overrides: "{{ _storage_overrides[meteringconfig_default_values | combine(meteringconfig_spec_overrides) | json_query('storage.hive.type')] | default({}, true) }}"
 
-meteringconfig_spec: "{{ meteringconfig_default_values | combine(meteringconfig_default_image_overrides, meteringconfig_storage_overrides, meteringconfig_spec_overrides, recursive=True) }}"
+#
+# Enable TLS by default
+#
+meteringconfig_reporting_operator_server_ca_certificate: ""
```

Comment:
The name of this variable, and the ones below for reporting_operator probably need the word `presto` in them, because it's confusing since reporting-operator has it's own server CA cert for it's own API.